EXILED/Exiled.API/Extensions/RoleExtensions.cs

```diff
@@ -123,6 +123,48 @@ public static bool TryGetRoleBase<T>(this RoleTypeId roleType, out T roleBase)
         /// <returns>Returns whether <paramref name="roleType"/> is an <see cref="IFpcRole"/> or not.</returns>
         public static bool IsFpcRole(this RoleTypeId roleType) => roleType.GetRoleBase() is IFpcRole;
 
+        /// <summary>
+        ///  Checks if the role is an SCP role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is an SCP role.</returns>
+        public static bool IsScp(this RoleTypeId roleType) => roleType.GetTeam() == Team.SCPs;
+
+        /// <summary>
+        /// Checks if the role is a dead role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is a dead role.</returns>
+        public static bool IsDead(this RoleTypeId roleType) => roleType.GetTeam() == Team.Dead;
+
+        /// <summary>
+        /// Checks if the role is an NTF role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is an NTF role. Does not include Facility Guards.</returns>
+        public static bool IsNtf(this RoleTypeId roleType) => roleType.GetTeam() == Team.FoundationForces && roleType != RoleTypeId.FacilityGuard;
+
+        /// <summary>
+        /// Checks if the role is a Chaos role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is a Chaos role.</returns>
+        public static bool IsChaos(this RoleTypeId roleType) => roleType.GetTeam() == Team.ChaosInsurgency;
+
+        /// <summary>
+        /// Checks if the role is a military role (Chaos Insurgency or NTF).
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is a military role. Does not include Facility Guards.</returns>
+        public static bool IsMilitary(this RoleTypeId roleType) => roleType.IsNtf() || roleType.IsChaos();
```

Comment:
Just does 
Ismilitary(this RoleTypeId roletype) =>  !roletype.IsCivilian()

I know this goes against what miki said but that what basegame believe for armor consumption of stamina for example

Reply:
Idk can we count tutorial as civilian (iirc when I checked it last time `RoleTypeId.Tutorial.IsCivilian()` will return false

Reply:
Tutorial is not civilian, no. so it will count as military, this change shouldn't be done @louis1706 as what I've done is perfect, this is just adding more work and more calculations for no reason.

Reply:
technically guard is a military class, I've said otherwise before because it doesn't make sense to me but that's how they're classified on the wiki, this method should reflect that

Reply:
Way to late to say that now @Mikihero 😭

Reply:
to make it work you can just add `|| roletype == RoleTypeId.FacilityGuard`, I don't think there really is a better solution since negating `IsCivilian` would return shit like tutorial etc.

Reply:
yeah, but this PR has been merged, if I could directly commit I would